sled-agent/src/rack_setup/service.rs

```diff
@@ -1121,8 +1144,10 @@ impl ServiceInner {
                 })?;
 
             info!(self.log, "advertising boundary services loopback address");
-            let mut ddmd_addr = dpd_addr;
-            ddmd_addr.set_port(8000);
+            // TODO - Should ddmd port be a constant?
```

Comment:
Yeah it probably should be.

Reply:
We define a lot of these ports in `omicron_common::address`, fwiw